scripts/kernel_build_conf.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This if fine for now, but you may need to move some of these values to the pipeline in the future.